wp-parsely.php

```diff
@@ -686,6 +686,8 @@ private function get_author_names($post) {
             $authors = array(get_user_by('id', $post->post_author));
         }
         $authors = array_map(array($this, 'get_author_name'), $authors);
+        $authors = array_map(array($this, 'get_author_name'), $authors);
```

Comment:
Whoops, looks like we now have a duplicated `$authors = array_map(array($this, 'get_author_name'), $authors);` line @adamsilverstein.